packages/react-core/src/components/Button/Button.tsx

```diff
@@ -64,6 +69,8 @@ export const Button: React.FunctionComponent<ButtonProps> = ({
   isBlock = false,
   isDisabled = false,
   isAriaDisabled = false,
+  isLoading = null,
+  spinnerAriaValueText,
```

Comment:
could set this to a reasonable default, like `Loading...` maybe

Reply:
@jessiehuff thoughts on the default value here.  Also does this prop name follow our conventions?

Reply:
So from consumer perceptive - we never use the default text values because their are not internationalized.
I guess everyone using PF also supports different languages so I am not convinced about the necessity of default text values.

Reply:
Regarding the property name it's aready used in the FileUploadField component.

Reply:
We already typically default to English values, in this case even if the value is not set and therefore undefined, Spinner defaults it already to "Loading..."

Reply:
So either way it doesn't matter too much in this PR